packages/a11y-testing/src/definitions/Button/buttonBehaviorDefinition.ts

```diff
@@ -27,8 +27,9 @@ export const buttonBehaviorDefinition: Rule[] = [
     .forProps({ disabled: true, loading: true })
     .doesNotHaveAttribute('disabled'),
   BehaviorRule.root()
-    .forProps({ disabled: true })
-    .hasAttribute('aria-disabled', 'true'),
+    .forProps({ disabledFocusable: true })
+    .hasAttribute('aria-disabled', 'true')
+    .doesNotHaveAttribute('disabled'),
```

Comment:
I see that `disabledFocusable` is tested here, while `Button-test` only test `disabled`. Seems inconsistent to me, either we test these properties in one place, or both places.

Also I think the combination of `loading` and `disabledFocusable` is not tested

Reply:
I think this is my missing work, that I overlooked remove accessibility tests from `Button-test`:
https://github.com/microsoft/fluentui/pull/15040/files
and I think we should keep all tests only in this `buttonBehaviorDefinition.ts` file.
I can do it as follow up PR, if Juraj think this is correct way to go.

Reply:
I will remove them, good catch

Reply:
I am planning to create a separate PR to disconnect loading and disabling as I think it is magic